AdapterInterface.php

```diff
@@ -75,7 +75,7 @@ public function compareAutoRouteContent(AutoRouteInterface $autoRoute, $contentO
      *
      * @return null|Symfony\Cmf\Component\Routing\RouteObjectInterface
      */
-    public function findRouteForUri($uri);
+    public function findRouteForUri($uri, UriContext $uriContext);
```

Comment:
here, UriContext is second argument, in createAutoRoute it is first. should the context always be last? or always first? and do you not have the getUri mismatch issue in createAutoRoute as well?